cmd/kubestellar-where-resolver/options/options.go

```diff
@@ -25,10 +25,16 @@ import (
 	clientoptions "github.com/kubestellar/kubestellar/pkg/client-options"
 )
 
+const (
+	defaultSpaceProviderName string = "default"
+	defaultKcsName           string = "espw"
+)
+
 type Options struct {
-	EspwClientOpts clientoptions.ClientOpts
-	BaseClientOpts clientoptions.ClientOpts
-	Logs           *logs.Options
+	SpaceMgtOpts  clientoptions.ClientOpts
```

Comment:
These are options not for "space management" in the full generality of that phrase, they are specifically for the kube client config that gives access to the space that holds the space management API objects. Perhaps a name like "SpaceMgtClientOpts" would be better?

Reply:
done